EIPS/eip-4844.md

```diff
@@ -45,10 +45,10 @@ Compared to full data sharding, this EIP has a reduced cap on the number of thes
 | `BLOB_COMMITMENT_VERSION_KZG` | `Bytes1(0x01)` |
 | `POINT_EVALUATION_PRECOMPILE_ADDRESS` | `Bytes20(0x14)` |
 | `POINT_EVALUATION_PRECOMPILE_GAS` | `50000` |
-| `MAX_DATA_GAS_PER_BLOCK` | `2**19` |
-| `TARGET_DATA_GAS_PER_BLOCK` | `2**18` |
+| `MAX_DATA_GAS_PER_BLOCK` | `786432` |
+| `TARGET_DATA_GAS_PER_BLOCK` | `393216` |
 | `MIN_DATA_GASPRICE` | `1` |
-| `DATA_GASPRICE_UPDATE_FRACTION` | `2225652` |
+| `DATA_GASPRICE_UPDATE_FRACTION` | `3338477` |
```

Comment:
why did this change?

Reply:
ah I see below this follows from the gas price sensitivity calculation in the "Data gasprice update rule" section below

Reply:
Exactly. I added a sentence pointing that out to the PR description.